model/model.go

```diff
@@ -13,12 +14,32 @@ type Event struct {
 }
 
 type Team struct {
-	Id               string
-	EventTag         string
+	Id               uint		//DB Primary key
+	Tag				 string
+	EventId 		 uint		//DB Primary key of the event
```

Comment:
Indentation :)

Reply:
is there something for automatical indentation ?

Reply:
Yeah, I actually pointed this one in next comment :)